components/layout_thread_2020/dom_wrapper.rs

```diff
@@ -108,6 +108,8 @@ pub struct ServoLayoutNode<'a> {
     /// Being chained to a PhantomData prevents `LayoutNode`s from escaping.
     chain: PhantomData<&'a ()>,
 }
+unsafe impl Send for ServoLayoutNode<'_> {}
+unsafe impl Sync for ServoLayoutNode<'_> {}
```

Comment:
I there a comment we could add here about the soundness of these impls?

Reply:
I added a comment.